src/map/clif.c

```diff
@@ -15876,7 +15876,7 @@ static void clif_parse_Mail_getattach(int fd, struct map_session_data *sd)
 	if( sd->mail.inbox.msg[i].zeny < 1 && (sd->mail.inbox.msg[i].item.nameid < 1 || sd->mail.inbox.msg[i].item.amount < 1) )
 		return;
 
-	if( sd->mail.inbox.msg[i].zeny + sd->status.zeny > MAX_ZENY ) {
+	if( sd->mail.inbox.msg[i].zeny > MAX_ZENY - sd->status.zeny ) {
```

Comment:
I think for correct check need add this
```if (sd->status.zeny >= MAX_ZENY || sd->mail.inbox.msg[i].zeny > MAX_ZENY - sd->status.zeny) {```
because if change MAX_ZENY from INT_MAX to some small number can be issue

Reply:
can sd->status.zeny be > MAX_ZENY at any moment?
It can be MAX_ZENY, but that would result in 0 in second condition, which would not cause any issue